src/ipa/ipu3/ipu3.cpp

```diff
@@ -265,7 +265,7 @@ class IPAIPU3 : public IPAIPU3Interface
 };
```

Comment:
This series jumps around between adding documentation to the IPAIPU3 class, the AWB/AGC/Tonemapping.

Try to keep your series' organised and grouped.

Patches dealing with the IPAIPU3 should keep the prefix
ipa: ipu3:

Patches modifying the AWB would use:
ipa: ipu3: awb:

and ...
ipa: ipu3: agc:
ipa: ipu3: tonemapping: 

accordingly.
then the patches should be ordered.

Reply:
Again, this one declares the IPAIPU3 as a reference design.
We shouldn't say that in the commits.